turbinia/turbiniactl.py

```diff
@@ -206,7 +206,7 @@ def main():
       'with. If this is different from the project that Turbinia is running '
       'in, it will be copied to the Turbinia project.')
   parser_googleclouddisk.add_argument(
-      '-P', '--mount_partition', default=0, type=int,
+      '--mount_partition', default=1, type=int,
```

Comment:
Why did the short flag (`-P`) get removed here?

Reply:
Looks like that's true for some of the flags below as well.

Reply:
That's weird. Fixed here and the other -P, though I didn't see any other case